packages/swapper/src/swappers/thorchain/utils/bitcoin/utils/getThorTxData.ts

```diff
@@ -0,0 +1,96 @@
+import { btcChainId } from '@shapeshiftoss/caip'
+import { bitcoin } from '@shapeshiftoss/chain-adapters'
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { Asset, BIP44Params, UtxoAccountType } from '@shapeshiftoss/types'
+
+import { SwapError, SwapErrorTypes } from '../../../../../api'
+import { bn, bnOrZero, fromBaseUnit, toBaseUnit } from '../../../../utils/bignumber'
+import { InboundResponse, ThorchainSwapperDeps } from '../../../types'
+import { getPriceRatio } from '../../getPriceRatio/getPriceRatio'
+import { makeSwapMemo } from '../../makeSwapMemo/makeSwapMemo'
+import { thorService } from '../../thorService'
+
+export const getThorTxInfo = async ({
+  deps,
+  sellAsset,
+  buyAsset,
+  sellAmount,
+  slippageTolerance,
+  destinationAddress,
+  wallet,
+  bip44Params,
+  accountType
+}: {
+  deps: ThorchainSwapperDeps
+  sellAsset: Asset
+  buyAsset: Asset
+  sellAmount: string
+  slippageTolerance: string
+  destinationAddress: string
+  wallet: HDWallet
+  bip44Params: BIP44Params
```

Comment:
do we need to pass these in?

https://github.com/shapeshift/lib/blob/main/packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts#L44